addons/point_of_sale/models/pos_order.py

```diff
@@ -1192,6 +1192,8 @@ def get_sale_details(self, date_start=False, date_stop=False, config_ids=False,
 
         if (session_ids):
             domain = AND([domain, [('session_id', 'in', session_ids)]])
+            date_start = self.env['pos.session'].browse(session_ids).start_at
```

Comment:
Why do you do that? I don't understand the goal of that, as when we specify a session, we want all of the session, so no need of start and stop date.

Reply:
Yeap, I agree with you on this. I made this part look confusing. So the reason I did this change is because when we the user printed a saleDetailsReport based on the session_id (our use case) the start date & end date placeholders were not initialized/missing, so the report would print something like this: <blank> - <blank>.

I thought it's preferable to initialize the date_start/end field so that the user is still informed about the time duration of the session. Should we drop it?

In case the session is still open we could add a 'session still open' next to the date_stop (current date) to make it more clear (or even completely replace the date_stop with the text). What do you think? Thank you!